docs/06_how-to-guides/40_multi-index/how-to-define-a-singleton.md

```diff
@@ -2,56 +2,90 @@
 content_title: How to define a singleton
 ---
 
-To define a simple singleton, which is storing an account name as primary value and a uint64_t as secondary value in structure `testtable`, follow the steps below:
+## Overview
 
-1. Include the `eosio.hpp` and `singleton.hpp` headers and declare the `eosio` namespace usage
-```
-#include <eosio/eosio.hpp>
-#include <eosio/singleton.hpp>
-using namespace eosio;
-```
+This guide provides instructions on how to define a singleton.
```

Comment:
Would be good to have a brief definition of a singleton and an example of where/why you mau use one.